simulators/lumi/qiskit/README.md

```diff
@@ -0,0 +1,51 @@
+# Running Quantum programs with Qiskit Simulator on LUMI
+
+## Loading the qiskit module
+The qiskit module on LUMI can be loaded using the following command:
+
+```
+module use /appl/local/quantum/modulefiles
+module load qiskit
+```
+
+This will load the default version of qiskit along with other major Qiskit packages (Terra, Nature, Aer, etc.)
```

Comment:
Let's describe which versions of Qiskit are currently on LUMI and how they are GPU accelerated.

Reply:
Yes please!